dali/ft/filecopy.cpp

```diff
@@ -1793,12 +1795,18 @@ void FileSprayer::gatherFileSizes(bool errorIfMissing)
 
 void FileSprayer::afterGatherFileSizes()
 {
-
     if (!copyCompressed)
     {
+        StringBuffer dateStr, timeStr, tailStr;
         ForEachItemIn(idx2, sources)
         {
             FilePartInfo & cur = sources.item(idx2);
+
+            LOG(MCdebugProgress, job, "%9d:%s (size: %llu bytes, last modified: %s %s)",
+                                       idx2, cur.filename.getTail(tailStr.clear()).str(), cur.size,
+                                       cur.modifiedTime.getDateString(dateStr.clear(), true).str(),
+                                       cur.modifiedTime.getTimeString(timeStr.clear(), true).str()
```

Comment:
CDateTime::getString() would do in 1 call and is slightly more efficient.

Reply:
Changed